src/pbs.rs

```diff
@@ -0,0 +1,45 @@
+use reqwest::{header, Client, ClientBuilder};
+use serde::{Deserialize, Serialize};
+use std::env;
+
+#[derive(Serialize, Deserialize, Debug)]
+pub struct DatastoreUsuage {
+    pub avail: i64,
+    #[serde(alias = "estimated-full-date")]
+    pub estimated_full_date: i64,
+    pub store: String,
+    pub total: i64,
+    pub used: i64,
+}
+
+#[derive(Serialize, Deserialize, Debug)]
+pub struct StatusResponse {
+    pub data: Vec<DatastoreUsuage>,
+}
+
+pub fn create_client() -> Client {
+    let token_id = env::var("TOKEN_ID").expect("$TOKEN_ID is not set");
```

Comment:
Sowas würde ich eigentlich in der Main Methode machen oder in einem extra config schritt und dann die config durch reichen. Geht aber auch so